tests/python/test_pyintegration.py

```diff
@@ -176,6 +177,9 @@ def test_annotations(self):
         with io.open(example_path, encoding="utf8") as example_handle:
             example_file = example_handle.read()
 
+        assert "software = sphinx" in example_file
+        assert "code_snippet = Multiline-String" in example_file
```

Comment:
For code owners: Is `Multiline-String` a good annotation name to use? Suggestions?